x/stakeibc/keeper/callbacks.go

```diff
@@ -9,28 +9,28 @@ import (
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	"github.com/spf13/cast"
 
-	"github.com/Stride-Labs/stride/x/interchainquery/types"
 	icqtypes "github.com/Stride-Labs/stride/x/interchainquery/types"
+	"github.com/Stride-Labs/stride/x/stakeibc/types"
 )
 
 // ___________________________________________________________________________________________________
 
 // Callbacks wrapper struct for interchainstaking keeper
-type Callback func(Keeper, sdk.Context, []byte, types.Query) error
+type Callback func(Keeper, sdk.Context, []byte, icqtypes.Query) error
```

Comment:
wdyt about renaming this file to `icqcallbacks` and renaming this struct accordingly?

Reply:
it's a good idea - going to leave it out of this PR in the interest of launching tomorrow (but added a note to do this EOD tomorrow)